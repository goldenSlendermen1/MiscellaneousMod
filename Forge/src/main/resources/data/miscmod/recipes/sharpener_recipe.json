{
  "type": "minecraft:crafting_shaped",
  "group": "functional",
  "category": "equipment",
  "pattern": [
    "aba",
    "aca"
  ],
  "key": {
    "a": {
      "item": "minecraft:iron_ingot"
    },
    "b": {
      "item": "miscmod:sawblade"
    },
    "c": {
      "item": "minecraft:deepslate"
    }
  },
  "result": {
    "item": "miscmod:sharpener",
    "count": 1
  }
}