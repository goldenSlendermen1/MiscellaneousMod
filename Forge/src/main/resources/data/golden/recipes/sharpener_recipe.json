{
  "type": "minecraft:crafting_shaped",
  "group": "functional",
  "category": "equipment",
  "pattern": [
    "aba",
    "aca"
  ],
  "key": {
    "a": {
      "item": "minecraft:iron_ingot"
    },
    "b": {
      "item": "golden:sawblade"
    },
    "c": {
      "item": "minecraft:deepslate"
    }
  },
  "result": {
    "item": "golden:sharpener",
    "count": 1
  }
}